src/Core/Content/Media/DataAbstractionLayer/MediaIndexer.php

```diff
@@ -97,16 +105,72 @@ public function handle(EntityIndexingMessage $message): void
             return;
         }
 
+        $context = $message->getContext();
+
+        $this->updateThumbnailsPath($context, $ids);
```

Comment:
Can you allow skipping this behavior?

Reply:
one skip for both is enough (updateThumbnailsPath and setMediaPaths)?

Reply:
Is it really worth to add this skipable? I mean, thumbnailRo is written anyway.